src/collections/blog/2022/2022-06-13-aaditya-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "My Journey as a Contributor to GSoC Intern @Layer5"
+subtitle: "Nothing but a magical experience"
+date: 2022-06-13 08:00:00 -0630
+author: Aaditya Narayan Subedy
+thumbnail: ./aaditya-subedy-layer5-intern.png
+category: Community
+description: "Aaditya Narayan Subedy's journey from a Contributor @Layer5 to getting an GSoC internship @Layer5"
+tags:
+ - Community
+ - Projects
+ - News
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+
+
+
+<BlogWrapper>
+
+### Early Interest in Coding
+
+I grew up with an immense interest in computers. I used to be fascinated by how programs run on the computer that makes such an impact on the people using it. This always motivated me to work in the field of computers.
+
+As I grew up I was introduced to a broad spectrum of coding languages that enabled us to make the software that we used on the computer. These languages piqued my interest even more, and I started to explore and make projects. The more I understood them the more interesting it got. Slowly I realized that I just don’t want to make projects that satisfy my needs. I want to build something that will be on the internet and will be used by people around the world. With this goal in my mind, I started exploring and hit a gemstone that is Open source. I would be working on real-world programming solutions that will be used by people all around the world. This helped me to get started and got me excited about contributing to open-source projects.
+
+### First PR @ Layer5
+
+I still remember the first issue I worked on for Layer5 was the auto labeler on a pull request. I saw a good first issue tag and still not knowing what the task was, I stated my interest to work on the issue. The issue required knowledge of GitHub Actions which at that time I didn’t even know there was such a thing. This is one amazing thing about open source, there is a community that is always there to help you. After doing some self-research and asking folks in the community I was finally able to solve the issue. I still remember the excitement when my PR got merged. I was enthralled to see that my code was actually working and being useful to the community. This motivated me to make more and more contributions.
```

Comment:
You can add a section for tips to make your way through the first contribution, even when you're doing something new. Breaking up the para and conveying all the steps you took in detail.